language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/MethodCallExpressionNodeContextTest.java

```diff
@@ -0,0 +1,30 @@
+package org.ballerinalang.langserver.completion;
+
+import org.ballerinalang.langserver.commons.workspace.WorkspaceDocumentException;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+/**
+ * Method call expression context related tests.
```

Comment:
Add the version

Reply:
Removed this calls and used `ExpressionContextTest` class.